src/components/common/YourCompany/EntityName.vue

```diff
@@ -323,6 +325,25 @@ export default class EntityName extends Mixins(CommonMixin, NameRequestMixin) {
     )
   }
 
+  /** Get the numbered name, after changing from named company */
+  getUpdatedName (): string {
+    if (this.getEntityType === CorpTypeCd.BC_ULC_COMPANY ||
+    this.getEntityType === CorpTypeCd.ULC_CONTINUE_IN) {
+      return 'UNLIMITED LIABILITY COMPANY'
+    }
+    if (this.getEntityType === CorpTypeCd.BC_CCC ||
+    this.getEntityType === CorpTypeCd.CCC_CONTINUE_IN) {
+      return 'COMMUNITY CONTRIBUTION COMPANY LTD.'
+    }
+    if (this.getEntityType === CorpTypeCd.BC_COMPANY ||
+    this.getEntityType === CorpTypeCd.CONTINUE_IN ||
+    this.getEntityType === CorpTypeCd.BENEFIT_COMPANY ||
+    this.getEntityType === CorpTypeCd.BEN_CONTINUE_IN) {
+      return 'LTD.'
+    }
+    return 'LTD.' // should never happen
+  }
```

Comment:
This is almost a repeat of the method in CorrectNameToNumber. It would be nice to have just one method. Thoughts?

(This isn't worth a lot of time to fix. Is there an easy change?)

Reply:
Moved the method to store.
For both components, can use param "this.getEntityType" to determine what to display